src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -32,22 +33,11 @@ class FileList extends React.Component {
     super(props);
 
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.damFileList = React.createRef();
```

Comment:
@digitalartcrew id probably just call this fileList, the class name has dam in it because css is global and whatnot